relayer-cli/src/commands/tx/transfer.rs

```diff
@@ -120,6 +121,49 @@ impl Runnable for TxIcs20MsgTransferCmd {
             Err(e) => return Output::error(format!("{}", e)).exit(),
         };
 
+        // Double check that channels and chain identifiers match.
+        // To do this, fetch from the source chain the channel end, then the associated connection
+        // end, and then the underlying client state; finally, check that this client is verifying
+        // headers for the destination chain.
+        let channel_end = src_chain
+            .query_channel(
+                &opts.packet_src_port_id,
+                &opts.packet_src_channel_id,
+                Height::zero(),
+            )
+            .unwrap_or_else(exit_with_unrecoverable_error);
+        // TODO: Support for multi-hop channels will impact this.
+        let conn_id = match channel_end.connection_hops.first() {
+            None => {
+                return Output::error(format!(
+                    "Could not retrieve the connection hop underlying channel end {:?} on chain {}",
```

Comment:
```suggestion
                    "could not retrieve the connection hop underlying channel end {:?} on chain {}",
```
Also in other places. I prefer errors and logs to start lower case. Up to you for this one.

Reply:
done for the log messages included in this PR! e7944169

ACK, will keep in mind the lower-case approach for future.